src/player.js

```diff
@@ -163,22 +163,29 @@ class Player {
      * Get a promise for a method.
      *
      * @param {string} name The API method to call.
-     * @param {Object} [args={}] Arguments to send via postMessage.
+     * @param {...(string|number|object|Array)} args Arguments to send via postMessage.
      * @return {Promise}
      */
-    callMethod(name, args = {}) {
+    callMethod(name, ...args) {
+        if (name === undefined || name === null) {
+            throw new TypeError('You must pass a method name.');
```

Comment:
Is this handling an existing issue, or is this more preemptive?

Reply:
It's more preemptive.